inspirehep/modules/crossref/core.py

```diff
@@ -56,3 +57,12 @@ def get_json(doi):
     data['query'] = query
 
     return data
+
+
+def normalize_doi(doi):
```

Comment:
actually, I just found out about `idutils.normalize_doi` which does the right thing.